client/src/pages/Login.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import Joi from 'joi';
+
+import axios from 'axios';
+import {
+  Avatar,
+  Button,
+  CssBaseline,
+  TextField,
+  Link,
+  Grid,
+  Box,
+  Typography,
+  Container,
+  FormControlLabel,
+  Checkbox,
+} from '@mui/material';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+
+const Copyright = (props: any) => {
+  return (
+    <Typography
+      variant="body2"
+      color="text.secondary"
+      align="center"
+      {...props}
+    >
+      {'Copyright © '}
+      <Link color="inherit" href="https://mui.com/">
+        Your Website
+      </Link>{' '}
+      {new Date().getFullYear()}.
+    </Typography>
+  );
+};
+
+const serverUrl = import.meta.env.VITE_APP_SERVER_URL;
+
+const Login = () => {
+  const [errors, setErrors] = React.useState({});
```

Comment:
You only use a single error, so name the state as error, not errors.
And the type for any error is `unknown` or `any`, and `any` is preferable. set the type for the state to be any and remove the object brackets argument.

Reply:
done 😎